rest-api-spec/src/main/resources/rest-api-spec/api/security.bulk_update_api_keys.json

```diff
@@ -0,0 +1,32 @@
+{
+  "security.bulk_update_api_keys": {
```

Comment:
@elastic/es-clients I was hoping to get your take on whether to use `bulk_update_api_keys` here or `update_api_keys`. 

I'm using `_bulk` to be consistent with the REST route path `.../_bulk_update` as well as better readability, to distinguish between `bulk_update_api_keys` and the single operation `update_api_key`. 

`update_api_key` and `update_api_keys` are quite close, and easy to misread. Since we want to be consistent between docs and the REST spec, this also extends to which API doc header we use, i.e. "Bulk update API keys" vs "Update API keys" and the same readability concern exists there.

The downside of `_bulk` is that we don't do this for other APIs, such as invalidate API keys.

I don't feel strongly either way, and was curious on your thoughts!

cc @ywangd

Reply:
I'm good with using "bulk" on the API endpoint name.

Btw, the `@elastic/es-clients` handle isn't used anymore. We use `@elastic/clients-team` now.

Reply:
I think we need a consistent strategy on using `bulk` (or not) moving forward. For example, if we introduce a new API in the future that does not have a counterpart for single operation, do we still want to have "bulk" in its name? Also, if we decide to deprecate (and evetually remove) a single operation API (maybe because it is entirely replaceable by the bulk version), this would leave a bulk-named API without the corresponding version of single operation. Do we care? @sethmlarson

Reply:
> Btw, the @elastic/es-clients handle isn't used anymore. We use @elastic/clients-team now.

Roger that! 

@ywangd thanks for raising this. I'll start a slack thread so we can collect more input.